### PR TITLE
[fix] KIC table Update

### PR DESCRIPTION
**Pull request by Guaris**
[[FTI-3221]](https://konghq.atlassian.net/browse/FTI-3221)

Reorders elements in the table, rename the section to Version Support for Kong Ingress Controller (Enterprise). 

Also adds sentence case to titles. 

[FTI-3221]: https://konghq.atlassian.net/browse/FTI-3221?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: app/konnect-platform/support-policy.md [hunk 1]
```diff
@@ -2,11 +2,12 @@
 title: Version Support Policy
```
Comment (cloudjumpercat) [inline app/konnect-platform/support-policy.md:2]: Should this be "Version Support for Kong Ingress Controller (Enterprise)"

**Review by cloudjumpercat — COMMENTED** [review]
Just one comment on the page title

File: app/konnect-platform/support-policy.md (hunk 1 above)
Comment (Guaris) [reply app/konnect-platform/support-policy.md:2]: Only line 83 is the Kong Ingress Controller table. Previously it was titled something differently. The doc is our product version support policy doc.

**Review by cloudjumpercat — APPROVED** [review]
LGTM!

**Pull request merged**